src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -828,6 +829,80 @@ public static BinaryOperandsKind GetBinaryOperandsKind(BinaryOperationKind kind)
         }
     }
 
+    public interface IConversion
```

Comment:
After looking at the C# and VB Conversion type, I understand that this approach is prior art. An IConversion interface that represents a subset of the properties of the preexisting types is a good thing.